tests/devices/test_default_qubit_tf.py

```diff
@@ -352,7 +352,7 @@ def test_invalid_state_prep_size(self):
         dev = DefaultQubitTF(wires=2)
         state = np.array([0, 1])
 
-        with pytest.raises(ValueError, match=r"State vector must have shape \(2\*\*wires,\)"):
+        with pytest.raises(ValueError, match=r"State must be of length 4"):
```

Comment:
Why this is changed?

Reply:
Same as before. The error occurs for the same reason with both templates but I had to choose one of the two default messages when unifying the templates.